src-electron/upgrade/upgrade.js

```diff
@@ -39,15 +46,53 @@ const fsp = fs.promises
 async function getForcedExternalStorage(db, attributeId) {
   let pkgs = await queryPackage.getPackageRefByAttributeId(db, attributeId)
   let zcl = await queryPackage.getPackageByPackageId(db, pkgs)
-  zcl = zcl.path
```

Comment:
This feels completely wrong. You are reading a JSON file that is part of the package. All those files should be loaded in during loading time, not here.... But I'd have to see what this does.

Reply:
as discussed, we will fix in follow up PR